experimenter/tests/integration/legacy/test_filter_expression_validation.py

```diff
@@ -1,14 +1,29 @@
-def test_filter_expressions_with_matching_firefox_versions(base_url, selenium):
+from pathlib import Path
+
+import pytest
+
+
+@pytest.fixture
+def filter_expression_path():
+    here = Path(__file__).cwd()
```

Comment:
This is wrong.

`Path.cwd()` returns the current working directory. `Path(xyz).cwd()` will first create a path pointing to `xyz` and then throw it out and return the current working directory.

If you want the parent directory of `__file__`, its `Path(__file__).parent()`.